Modules/Finance/SubModules/Export.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ConnectWise.Http.ModuleTypes;
+
+namespace ConnectWise.Http.Modules.Finance.SubModules
+{
+	public class ExportSubModule: PartialSubModule
```

Comment:
so since this submodule actually starts with `/accounting` and the endpoint is `/export` this should be called an `AccountingSubModule` - following how the rest are named.

Also since it inherits from `PartialSubModule` it is implementing a couple of endpoints that don't exist in the REST API, The `PartialSubModule` implements the `CREATE` & `DELETE` endpoints that you don't want here so you probably just want to inherit from `SubModule` itself since `/export` is the only endpoint you want.

Looks like there is a bunch of extra whitespace here too.

Reply:
ok, changed but brother, I don't have the whitespace detector. :)  Not sure what you mean by that.